docs/source/developer/interactives.rst

```diff
@@ -112,32 +49,53 @@ Every interactive will follow a very similar file structure as given below:
       ├── static/
       │   ├── interactives/
       │   │   └── <interactive-slug>/
-      │   │       ├── css/
+      │   │       ├── scss/
       │   │       │   ├── third-party/
-      │   │       │   └── style.css
+      │   │       │   └── <interactive-slug>.scss
       │   │       ├── img/
       │   │       ├── js/
       │   │       │   ├── third-party/
-      │   │       │   └── script.js
-      │   │       ├── README.md
-      │   │       └── thumbnail.png
+      │   │       │   └── <interactive-slug>.js
+      │   │       ├── package.json
+      │   │       └── README.md
       │   ├── js/
       │   │   └── third-party/
-      │   └── css/
+      │   └── scss/
       │       └── third-party/
       └── templates/
           └── interactives/
-              └── <interactive-template>.html
+              └── <interactive-slug>.html
 
 .. note::
 
   Each interactive has a unique ``slug``, which is used in file/directory names and urls. For example the slug for the ``High Score Boxes`` interactive is ``high-score-boxes``.
 
 Items of interest from this diagram:
 
-- ``static/interactives/<interactive-slug>/`` - This is the directory where static files for the interactive are stored, which should be broken down into ``css``, ``js`` and ``img`` directories.
+- ``static/interactives/<interactive-slug>/`` - This is the directory where static files for the interactive are stored, which should be broken down into ``scss``, ``js`` and ``img`` directories.
+
+- ``static/interactives/<interactive-slug>/package.json`` - The ``package.json`` file is npm's configuration file for projects and modules.
```

Comment:
Add something like; `The file should only be included if the interactive loads at least 1 npm module`